jetty-webapp/src/main/java/org/eclipse/jetty/webapp/MetaInfConfiguration.java

```diff
@@ -618,6 +618,7 @@ public void postConfigure(WebAppContext context) throws Exception
         if (web_inf_lib.exists() && web_inf_lib.isDirectory())
         {
             String[] files=web_inf_lib.list();
+            // TODO should files be sorted? What is the interaction with Ordering?
```

Comment:
Did not manage to track down usage of these APIs, but I was pretty sure `WebAppClassLoader` was really used in a normal configuration, and it was definitely not sorting JARs.